config/flare.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+ /*
+ | !!!! DO NOT EDIT THIS FILE !!!!
+ |
+ | You can change settings by setting them in the environment or .env
+ | If there is something you need to change, but is not available as an environment setting,
+ | request an environment variable to be created upstream or send a pull request.
+ */
+
+return [
+    /*
+    |
+    |--------------------------------------------------------------------------
+    | Flare API key
+    |--------------------------------------------------------------------------
+    |
+    | Specify Flare's API key below to enable error reporting to the service.
+    |
+    | More info: https://flareapp.io/docs/general/projects
+    |
+    */
+
+    'key' => null,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Reporting Options
+    |--------------------------------------------------------------------------
+    |
+    | These options determine which information will be transmitted to Flare.
+    |
+    */
+
+    'reporting' => [
+        'anonymize_ips' => true,
+        'collect_git_information' => true,
+        'report_queries' => true,
+        'maximum_number_of_collected_queries' => 200,
+        'report_query_bindings' => true,
```

Comment:
You sure about query bindings?

Reply:
Think we should start with it and see if it contains too much sensitive info